<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoCell</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="numeroLabel">
              <property name="text">
               <string>Numéro</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="numeroInput">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="bitLabel1">
              <property name="text">
               <string>111</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="bitLabel2">
              <property name="text">
               <string>110</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="bitLabel3">
              <property name="text">
               <string>110</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="bitLabel4">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit4">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="bitLabel5">
              <property name="text">
               <string>011</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit5">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="bitLabel6">
              <property name="text">
               <string>010</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit6">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="bitLabel7">
              <property name="text">
               <string>001</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit7">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="bitLabel8">
              <property name="text">
               <string>000</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bit8">
              <property name="text">
               <string>0</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="grilleDepart">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPlaySimulation">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="grille">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir_un_Automate"/>
    <addaction name="actionEnregistrer_l_automate"/>
   </widget>
   <widget class="QMenu" name="menuAutomate">
    <property name="title">
     <string>Automate</string>
    </property>
    <addaction name="actionElementaire"/>
    <addaction name="actionGoL"/>
    <addaction name="actionWireworld"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAutomate"/>
  </widget>
  <action name="actionOuvrir_un_Automate">
   <property name="text">
    <string>Ouvrir un automate</string>
   </property>
  </action>
  <action name="actionEnregistrer_l_automate">
   <property name="text">
    <string>Enregistrer l'automate</string>
   </property>
  </action>
  <action name="actionElementaire">
   <property name="text">
    <string>Élémentaire</string>
   </property>
  </action>
  <action name="actionGoL">
   <property name="text">
    <string>Jeu de la Vie</string>
   </property>
  </action>
  <action name="actionWireworld">
   <property name="text">
    <string>Wireworld</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
